tools/devops/automation/templates/windows/build.yml

```diff
@@ -69,6 +69,32 @@ steps:
   displayName: 'Show .NET info'
   continueOnError: true
 
+- task: AzureKeyVault@2
+  inputs:
+    azureSubscription: 'Xamarin - R&D - XamarinSecurity'
+    KeyVaultName: 'xamarin-ios-vault'
+    SecretsFilter: 'RemoteMacIdRsa'
+  displayName: 'Download id_rsa'
+
+- pwsh: |
+    $idRsaPath = "$(Get-Location)\id_rsa"
```

Comment:
You could do the following in pwsh:

```pwsh
Join-Path -Path $(Get-Location) -ChildPath "id_rsa"
```

And avoid all the weird things with paths.

Reply:
pwsh is quite verbose...